modules/gapi/src/backends/cpu/gcpustereo.cpp

```diff
@@ -0,0 +1,47 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#include "precomp.hpp"
+#include "gnnparsers.hpp"
+
+#include <opencv2/calib3d.hpp>
+
+#include <opencv2/gapi/stereo.hpp>
+#include <opencv2/gapi/cpu/stereo.hpp>
+#include <opencv2/gapi/cpu/gcpukernel.hpp>
+
+#include <iostream>
+
+GAPI_OCV_KERNEL(GCPUStereoBM, cv::gapi::stereo::GStereoBM)
+{
+    static void run(const cv::Mat& left,
+                    const cv::Mat& right,
+                    const int numDisparities,
+                    int blockSize,
+                    const cv::gapi::stereo::OutputFormat oF,
+                    const int dtype,
+                    const double baseline,
+                    const double focus, 
+                    cv::Mat& out_disp)
+    {
+        cv::Mat disp;
+        cv::StereoBM::create(numDisparities, blockSize) -> compute(left, right, disp);
+
+        // See https://answers.opencv.org/question/199915/how-to-get-depth-from-disparity-map/?sort=votes#:~:text=Z%20%3D%20B*f%20%2F%20disparity%20is%20the%20correct%20formula%20to,get%20the%20depth%20in%20mm.
```

Comment:
I believe such kind of reference shouldn't be there

Reply:
Removed